spring-grpc-spring-boot-autoconfigure/src/main/java/org/springframework/grpc/autoconfigure/server/GrpcServerReflectionAutoConfiguration.java

```diff
@@ -1,30 +1,29 @@
 package org.springframework.grpc.autoconfigure.server;
 
+import org.springframework.boot.autoconfigure.AutoConfiguration;
 import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
 
 import io.grpc.BindableService;
 import io.grpc.protobuf.services.ProtoReflectionService;
 
 /**
  * {@link EnableAutoConfiguration Auto-configuration} for gRPC Reflection service
  * <p>
- * grpc.reflection.enabled=true must be present in configuration in order for the
+ * spring.grpc.reflection.enabled=true must be present in configuration in order for the
```

Comment:
This is no longer quite true right? It’s on by default (and hence not needed explicitly in the samples)?

Reply:
True, will update soon!

Reply:
* Auto-configuration enabled by default
 Should I replace with this kind of comment or?

Reply:
Makes sense. You can still mention the flag, but use “false” ad the example because that’s likely what most people will need.

Reply:
/**
 * {@link EnableAutoConfiguration Auto-configuration} for gRPC Reflection service
 * <p>
 * This auto-configuration is enabled by default. To disable it, set the configuration
 * flag {spring.grpc.server.reflection.enabled=false} in your application properties.
 * auto-configuration to execute
 *
 * @author Haris Zujo
 */
 
 
 Updated